### PR TITLE
Single-multiple.md update

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
### Description
Edited single-multiple.md file to improve and clarify content, voice, tense, and grammar.
 
### Issues Resolved
Makes the content more usable.

### Check List
- [x ] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _installation/single-multiple.md [hunk 1]
```diff
@@ -10,66 +10,52 @@ Babelfish offers two modes of operation:
...
-collisions.
-
-The migration mode is stored in the `migration_mode` parameter. You can't change
-this parameter after creating your cluster (if you want to change the migration
-mode, you'll have to create a new cluster).
-During the deployment process you must decide which mode to use.  If you don't
-set `migration_mode`, the default value of `single-db` is chosen.
-
-How does this work? Consider the following code snippet:  
+When you create a Babelfish cluster, you choose between using a single T-SQL database or multiple T-SQL databases together. Your choice will affects how the names of T-SQL schemas inside the Babelfish database appear in PostgreSQL. For more information about selecting between `single-db` and `multi-db` mode, review [Choosing a migration mode](https://babelfishpg.org/docs/installation/single-multiple/#choosing-a-migration-mode).
```
Comment (rcv-aws) [inline _installation/single-multiple.md:13]: "single T-SQL database" ==> "single migrated T-SQL user database"
"multiple T-SQL databases together" ==> "multiple migrated T-SQL user databases together"

File: _installation/single-multiple.md [hunk 2]
```diff
@@ -10,66 +10,52 @@ Babelfish offers two modes of operation:
...
 
 
 ### Choosing a migration mode
 
 Each migration mode has advantages and disadvantages. Choose your migration
 mode based on the number of user databases you have, and your migration plans.
-After you create a cluster for use with Babelfish, you can't change the
+After you initialize Babelfish, you can't change the
 migration mode. When choosing a migration mode, consider the requirements of
 your user databases and clients.
```
Comment (rcv-aws) [inline _installation/single-multiple.md:54]: clients ==> client applications

File: _installation/single-multiple.md [hunk 3]
```diff
@@ -78,7 +64,7 @@ Use single database migration mode in the following cases:
 - If you are migrating a single SQL Server database. In single database mode,
   migrated schema names (when seen from PostgreSQL) are identical to the
   original SQL Server schema names.
-  In case you plan to ultimately migrate your application to native
+  If you plan to ultimately migrate your application to native
   PostgreSQL, your SQL code will require fewer modifications.
```
Comment (rcv-aws) [inline _installation/single-multiple.md:68]: will ==> may

**Review by rcv-aws — CHANGES_REQUESTED** [review]
added various comments

File: _installation/single-multiple.md (hunk 1 above)
Comment (susanmdouglas-aws) [reply _installation/single-multiple.md:13]: Updated, thanks!

File: _installation/single-multiple.md (hunk 2 above)
Comment (susanmdouglas-aws) [reply _installation/single-multiple.md:54]: Updated... thanks!

File: _installation/single-multiple.md (hunk 3 above)
Comment (susanmdouglas-aws) [reply _installation/single-multiple.md:68]: Updated, thanks!

**Comment by susanmdouglas-aws** [comment]
I've updated the file and checked in the updates... thanks!

**Review by 3manuek — DISMISSED** [review]

File: _installation/single-multiple.md [hunk 4]
```diff
@@ -10,66 +10,52 @@ Babelfish offers two modes of operation:
...
-collisions.
-
-The migration mode is stored in the `migration_mode` parameter. You can't change
-this parameter after creating your cluster (if you want to change the migration
-mode, you'll have to create a new cluster).
-During the deployment process you must decide which mode to use.  If you don't
-set `migration_mode`, the default value of `single-db` is chosen.
-
-How does this work? Consider the following code snippet:  
+When you create a Babelfish cluster, you choose between using a single migrated T-SQL user database or multiple migrated T-SQL user databases together. Your choice will affects how the names of T-SQL schemas inside the Babelfish database appear in PostgreSQL. For more information about selecting between `single-db` and `multi-db` mode, review [Choosing a migration mode](https://babelfishpg.org/docs/installation/single-multiple/#choosing-a-migration-mode).
```
Comment (rcv-aws) [inline _installation/single-multiple.md:13]: affects ==> affect

**Review by rcv-aws — APPROVED** [review]
just a grammar error, otherwise OK

File: _installation/single-multiple.md (hunk 4 above)
Comment (susanmdouglas-aws) [reply _installation/single-multiple.md]: Thank you - fixed!

**Review by rcv-aws — APPROVED** [review]

**Review by 3manuek — APPROVED** [review]

**Pull request merged**